packages/lib/date-fns/index.ts

```diff
@@ -6,3 +6,73 @@ export const yyyymmdd = (date: Date | Dayjs) =>
 
 export const daysInMonth = (date: Date | Dayjs) =>
   date instanceof Date ? dayjs(date).daysInMonth() : date.daysInMonth();
+
+/**
+ * Expects timeFormat to be either 12 or 24, if null or undefined
+ * is passed in, we always default back to 24 hour notation.
+ */
+export const formatTime = (
+  date: string | Date | Dayjs,
+  timeFormat?: number | null,
+  timeZone?: string | null
+) => {
+  const utcOffset = dayjs()
+    .tz(timeZone || undefined)
+    .utcOffset();
+
+  return timeZone
+    ? dayjs
+        .utc(date)
+        .utcOffset(utcOffset)
+        .format(timeFormat === 12 ? "h:mma" : "HH:mm")
+    : dayjs(date).format(timeFormat === 12 ? "h:mma" : "HH:mm");
+};
+/**
```

Comment:
```suggestion

/**
```
we add some space between functions here